github-actions-reporter.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-console */
```

Comment:
@gvamshi-metrological  Can you please remove this file, also because we can't use Github actions 😩